.github/workflows/tests.yml

```diff
@@ -16,11 +16,14 @@ jobs:
       - name: Install Dafny
         uses: dafny-lang/setup-dafny-action@v1
         with:
-          dafny-version: "3.4.2"
+          dafny-version: "3.9.1"
```

Comment:
I'd prefer that we don't do this until forced to. Ideally this code would be useable with the widest range of Dafny versions possible, especially since it's common for upgrades to be difficult due to changes in verification performance.

Reply:
Note that you can just change your LIT commands to use the old CLI syntax instead, e.g:

```
%dafny /compile:4 /noVerify /compileTarget:cs "%s" "%S/../../src/FileIO/FileIO.cs" /args "%S/data.txt" "System.ArgumentException:" >> "%t"
```

(Or close to that at least)

Reply:
As far as I can tell, v3.9.1 is the earliest version of Dafny under which these tests can reasonably run. That's because command-line arguments were added in v3.9.0, and they were [made consistent in JS](https://github.com/dafny-lang/dafny/pull/2876) in v3.9.1.

Reply:
Got it, that's a harder motivation to work around.

This is fine for now then. If and when we get requests to guarantee this code will work on older versions of Dafny, we can test on a wider range. We may even need to do that on a per "library" basis.